src/app/modules/shared_models/balance_model.nim

```diff
@@ -75,8 +75,13 @@ QtObject:
       of "address": result = $item.address
       of "balance": result = $item.balance
 
+
+  proc cmpBalances*(x, y: Item): int =
+    cmp(x.balance.getAmount(), y.balance.getAmount())
```

Comment:
I often find myself caught off guard when I see a call to a comparison without a 'return' statement... but then I realize is Nim! 😄